src/samples/flow/c-api/lowlevel.c

```diff
@@ -142,7 +136,7 @@ startup(void)
     nodes[1 /* logic */].type = SOL_FLOW_NODE_TYPE_CUSTOM_NODE_TYPES_LOGIC;
     nodes[2 /* writer */].type = SOL_FLOW_NODE_TYPE_CUSTOM_NODE_TYPES_WRITER;
 #ifdef SOL_FLOW_NODE_TYPE_CONSOLE_DEFINED
```

Comment:
we don't need this `ifdef` here, right? Or this ifdef should also cover `#include "sol-flow/console.h"`